content/blaze.md

```diff
@@ -0,0 +1,543 @@
+---
+title: Blaze
+---
+
+After reading this guide, you'll know:
+
+1. How to use the Spacebars syntax, used to define Blaze templates.
```

Comment:
Hmm, technically they are Spacebars templates, Blaze is just the underlying rendering engine... what do you think? Too pedantic? People use the word "Blaze" to refer to the whole thing these days.